app/[slug]/page.tsx

```diff
@@ -193,14 +190,6 @@ export default function Group() {
           </Container>
         </Card.Body>
       </Card>
-    </Container>
+    </div>
```

Comment:
This needs to close off a `<Container>` from line [#35 here](https://github.com/Nola-Devs/Nola-Dev-Site/blob/main/app/%5Bslug%5D/page.tsx#L35) or line # 30 in your merge branch